opentreemap/treemap/lib/map_feature.py

```diff
@@ -282,6 +282,15 @@ def context_dict_for_resource(request, resource, **kwargs):
 
     _add_share_context(context, request, photos)
 
+    object_name_alias = to_object_name(context['feature'].__class__.__name__)
+    # some features that were originally written to support plot and tree
+    # have grown to support other resource types, but they expect a context
+    # entry for their type, not just for 'feature'.
+    # For example:
+    # * Plot detail expects 'plot' and 'tree'
+    # * Foo detail would expect 'foo'
+    context[object_name_alias] = context['feature']
```

Comment:
Should we ``del context['feature']`` to avoid confusion? Does that break template rendering expectations unnecessarily?